scout/adapter/mongo/variant.py

```diff
@@ -326,6 +339,32 @@ def gene_variants(
 
         return result
 
+    def count_gene_variants(
```

Comment:
Nice! Could you add a simple test for this one?

Reply:
Done!

Reply:
Done!